File: other/coding/FizzBuzz.js
void FizzBuzz(n) {
	for (var i = 1; i <= n; i++) {
		if ((i%5 != 0) && (1%3 != 0)) {console.log(i)}
		if (1%3 == 0) {console.log("Fizz")}
		if (1%5 == 0) {console.log("Buzz")}
	}
}

FizzBuzz(100);

File: other/coding/FizzBuzz.py
for x in range(1,100):
	if ((x%3!=0) and (x%5!=0)) :
		print(x, end='');
	if (x%3==0) :
		print("Fizz", end='');
	if (x%5==0) :
		print("Buzz", end='');
	print('');
import sys
print(sys.version);
